spec/controllers/searches_controller_spec.rb

```diff
@@ -52,7 +52,8 @@
 
     context 'when searching in English (redesign)' do
       before do
-        get :index, params: { query: 'social security', affiliate: 'usagov', redesign: 'true' }
+        affiliate.update(use_redesigned_results_page: true)
```

Comment:
It'd be better to have an affiliate with use_redesigned_results_page set to true that we could reuse so we don't have to update it on every test. What do you think about adding a new fixture, something like:

```yml
# spec/fixtures/affiliates.yml

redesigned_usagov:
  <<: *DEFAULTS
  display_name: USA.gov
  name: new-usagov
  website: https://www.usa.gov
  search_engine: BingV7
  use_redesigned_results_page: true
```

and use it instead:

```rb
  let(:affiliate) { affiliates(:redesigned_usagov) }
```